### DISCUSSION TITLE
How to update to Yellow 0.6.1?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#121] [Ask a question] [answered]
As far as I can see, version 0.6.1 chanced the structure of config file. Instead of the equal sign as separator now a colon is used. This leads to several unspecific errors after updating a previous installation, for example wrong style-sheet, if not „default“ is used.

It would be nice, if such changes would be notified in readme file or version page.

**Comment by markseuffert** [comment, marked as answer]
[Yellow v0.6.1 was released today](https://github.com/datenstrom/yellow/releases). Here’s how to update an existing installation:
1. Update the `yellow.php` file [with new file](https://github.com/datenstrom/yellow/blob/master/yellow.php)
2. Update the `system/plugins` folder with new files
3. Delete the `system/core` folder, it's no longer needed
4. Go to the command line and run `php yellow.php version`, this will update the config files
5. Open your website and check that everything works

Sorry for any troubles and thanks for the report. Please let me know if this works.

**Comment by BsNoSi** [comment]
Everything´s fine! 

**Updating Yellow is really simple.** Because I do not have direct access to the command line on my servers (hosting packages), I have to do these changes manually. 

Basically updating it is no great challenge with Yellow. But "smart" chances like colon instead of equal sign in config.ini are quite tricky, if you do not know about. 

Therefore a short list of relevant modifications would help. **This is not an issue, it is a personal wish regarding documentation.**

As soon as I found the "problem" (I searched in the wrong places in the beginning), everything worked promptly and as expected.

**Comment by markseuffert** [comment]
Sorry about this, the changes should have been communicated before the release. I am not planning to make any structural changes in the near future, to be honest I hate these things and avoid them as long as possible. Good that your website is working again. Please give us feedback if something is not working, both problems and contributions are always welcome.

Thanks.

**Comment by BsNoSi** [comment]
Nothing was destroyed, nothing got lost. A very big advantage of this concept — one more reason for me to trust.

There is no reason to apologize. It took a little time to find the solution, fixing took seconds. I take it as chance, to get more familiar with Yellow. It is a "version zero", I do accept, that there are changes to make it better. I am sorry, that I have currently and in the near future no time to give more support to the project.
